_includes/panel_action_modal.html

```diff
@@ -106,46 +105,39 @@
           '<option' + field_value + subtext + disabled + '>' + legislator.name + '</option>'
         );
       });
-      $('.btn-group.bootstrap-select').hide()
+      $('.btn-group.bootstrap-select').remove()
       $nomineeSelect.removeClass('hidden').selectpicker();
     }
   });
 }
 
-
 $(document).ready(function(){
   fetch_legislator_info();
 
-  $(".show-nomination-modal").click(function() {
-    $("#js-nomination-form").trigger("reset");
+  $(".show-nomination-modal").on("click", function() {
+    $("textarea, select").val(function() {
+        return this.defaultValue;
+    });
     $("#additional-fields").show();
-    $("#nomination-page-2").hide();
+    $("#nomination-page-thanks").hide();
     $("#nomination-page-1").show();
     $("#submit-candidate-modal").modal("show");
     $(".bootstrap-select .filter-option").text("Search by name or state");
   });
 
-  $("#submit-again").click(function() {
-    $("#nomination-page-2").hide();
+  $("#submit-again").on("click", function() {
+    $("#nomination-page-thanks").hide();
     $("#nomination-page-1").show();
   });
 
   $("#js-nomination-form").submit(function (event) {
     event.preventDefault();
-
-    if ($("select[name=legislator_id]").val() == "" && $("input[name=email]").val() == "") {
-      $("button.selectpicker").focus();
-    } else {
-      var saveURI = '{{site.services_url}}/nominations';
-      $.post(saveURI, $(this).serialize());
-      $("#nomination-page-1").hide();
-      $("#nomination-page-2").show();
-      $("#additional-fields").hide();
-      $("textarea[name=selection_comment]").val("");
-      $("textarea[name=other_comment]").val("");
-      $("select[name=legislator_id]").val("")
```

Comment:
It does happen.  It happens before the form gets displayed again - right before the show is actually called.  I moved it to better associate the code to when it's actually required to be called.  This way if someone is quickly glacing they won't wonder why were clearing the form, but clearly see the related call to unhide it.

However, It actually seems like .val("") and what I wrote above are functionally equivalent.  I should just do `$("textarea, select").val("")`.  Will update later.